### PR TITLE
chore(alerts): migration UI cleanup

### PR DESCRIPTION
**Pull request by obostjancic**
- Part of #60356

### CODE REVIEW

File: static/app/views/alerts/edit.tsx [hunk 1]
```diff
@@ -76,7 +73,7 @@ class ProjectAlertsEditor extends Component<Props, State> {
           <Layout.HeaderContent>
             <BuilderBreadCrumbs
               organization={organization}
-              title={isMigration ? t('Review Thresholds') : t('Edit Alert Rule')}
```
Comment (scttcper) [inline static/app/views/alerts/edit.tsx:79]: we want to keep this for now

File: static/app/views/alerts/list/rules/alertRulesList.tsx [hunk 2]
```diff
@@ -207,17 +195,8 @@ function AlertRulesList() {
         <AlertHeader router={router} activeTab="rules" />
         <Layout.Body>
           <Layout.Main fullWidth>
-            {showMigrationUI ? (
```
Comment (scttcper) [inline static/app/views/alerts/list/rules/alertRulesList.tsx:210]: all this stuff is fine to remove

File: static/app/views/alerts/rules/metric/ruleConditionsForm.tsx [hunk 3]
```diff
@@ -407,170 +404,157 @@ class RuleConditionsForm extends PureComponent<Props, State> {
         <ChartPanel>
           <StyledPanelBody>{this.props.thresholdChart}</StyledPanelBody>
         </ChartPanel>
-        {isMigration ? (
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/ruleConditionsForm.tsx:410]: keep for now

File: static/app/views/alerts/rules/metric/ruleForm.tsx [hunk 4]
```diff
@@ -1076,16 +1060,13 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
               </Confirm>
             ) : null
           }
-          submitLabel={
-            isMigration && !triggersHaveChanged ? t('Looks good to me!') : t('Save Rule')
-          }
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/ruleForm.tsx:1081]: keep

File: static/app/views/alerts/rules/metric/ruleForm.tsx [hunk 5]
```diff
@@ -903,8 +894,7 @@ class RuleFormContainer extends DeprecatedAsyncComponent<Props, State> {
       location,
       query: this.chartQuery,
       aggregate,
-      // If the alert is being migrated, we want to use the generic metrics dataset to allow users to edit their thresholds
-      dataset: isTransactionMigration ? Dataset.GENERIC_METRICS : dataset,
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/ruleForm.tsx:907]: this looks fine to remove

File: static/app/views/alerts/rules/metric/triggers/index.tsx [hunk 6]
```diff
@@ -135,20 +133,18 @@ class Triggers extends Component<Props> {
           </PanelBody>
         </Panel>
 
-        {isMigration ? null : (
```
Comment (scttcper) [inline static/app/views/alerts/rules/metric/triggers/index.tsx:138]: keep

**Comment by obostjancic** [comment]
@scttcper Sorry, i was not aware of that. I updated this PR now removing on transaction migration logic, but pls check if everything is ok in the preview deployment. Thanks!

**Review by scttcper — APPROVED** [review]

**Comment by scttcper** [comment]
looks good thanks!

**Pull request merged**